cms/models/placeholdermodel.py

```diff
@@ -289,15 +282,12 @@ def _get_attached_model(self):
 
     def _get_attached_admin(self, admin_site=None):
```

Comment:
remove this method

Reply:
The save method of plugin base requires this:

    def save_model(self, request, obj, form, change):
        """
        Override original method, and add some attributes to obj
        This have to be made, because if object is newly created, he must know
        where he lives.
        """
        pl = obj.placeholder
        pl_admin = pl._get_attached_admin()